Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -1063,6 +1066,11 @@ extension TelemetryWrapper {
         // MARK: Start Search Button
         case (.action, .tap, .startSearchButton, _, _):
             GleanMetrics.Search.startSearchPressed.add()
+        case(.action, .tap, .recordSearch, _, let extras):
+            if let searchLocation = extras?[EventExtraKey.recordSearchLocation.rawValue] as? SearchesMeasurement.SearchLocation, let searchEngineID = extras?[EventExtraKey.recordSearchEngineID.rawValue] as? String? {
+                Telemetry.default.recordSearch(location: searchLocation, searchEngine: searchEngineID ?? "other")
+                GleanMetrics.Search.counts["\(searchEngineID ?? "custom").\(searchLocation.rawValue)"].add()
+            }
```

Comment:
Can we add an `else` case for `recordUninstrumentedMetrics` like we do in other places? This help us catch if we ever have a telemetry problem in this class.

Reply:
Hi @lmarceau I have added `recordUninstrumentedMetrics `